dht.go

```diff
@@ -93,6 +95,10 @@ type Config struct {
 	ThrottlerTrackedClients int64
 	//Protocol for UDP connections, udp4= IPv4, udp6 = IPv6
 	UDPProto string
+	// max get_peer requests per hash to prevent infinity loop
+	MaxSearchQueries int
+  // number of concurrent listeners on same port
+	ConnPoolSize int
```

Comment:
I don't think this will work? The code is not safe for concurrent use, right? If you want to use multiple goroutines you need different DHT instances

Reply:
as mentioned by email I broke my fork by commiting testing code and have no clue how to revert